#!/usr/bin/python3

def raise_exception_msg(message=""):
    """Raise a NameError exceptiosn with a message."""
    raise NameError(message)
